solucoes/desafio_01/lbighetti.exs

```diff
@@ -0,0 +1,14 @@
+# Desafio 01 grupo de estudos
+
+hints = "Poção mágica, liquido encantado em frasco."
+IO.puts "Dicas: #{hints}"
+
+guess = IO.gets "Adivinhe a palavra: "
+guess = String.trim(guess)
```

Comment:
E se a resposta for "Elixir"? Vai dar certo?

Reply:
Não.... 😛 
Vou dar um tapa, valeu!
man, você acha que podiamos colocar no requisito do desafio ?
funcionar independente de case?
ou deixa pra quem tiver implementando pensar mesmo ?

Reply:
acho válido deixar pra quem implementar pensar nisso. até pq na vida real frequentemente os requisitos vem incompletos :P